samples/IdentitySample.Mvc/Startup.cs

```diff
@@ -7,6 +7,10 @@
 using Microsoft.Framework.DependencyInjection;
 using Microsoft.Framework.Logging;
 using Microsoft.Framework.Logging.Console;
+#if DNX451
+using NLog.Config;
```

Comment:
Let's file a bug on our side for now (i.e. in Logging), so that we don't forget this is broken.